### ISSUE TITLE
Disallow "line break" in parent term

### ISSUE THREAD

**Issue by Mycheze** [jzohrab/lute-v3#299]
**Description**

Maybe this is just my weird corner case, but having linebreaks in the parent seems pointless. I frequently copy things to paste into the parent term, and if I move my mouse .05 px too far, it'll grab a linebreak and I have to manually delete it to have the search actually be successful/

**To Reproduce**

Steps to reproduce the behavior, e.g.:

1. Paste a parent term in which has a leading linebreak
(
parentterm)

**Screenshots**

If it will be helpful, add screenshots.

**Extra software info, if not already included in the Description:**

- OS (e.g., iOS, windows):
- Browser (e.g., chrome, safari):
- How you've installed Lute (Docker, python, source)
- Version:

**Comment by jzohrab** [comment]
Hola @Mycheze - confirming, do you mean something like this?

> <img width="90" alt="image" src="https://github.com/jzohrab/lute-v3/assets/1637133/d7d8532a-d3c1-4df0-b74f-f90bbc8c0db5">

I have "amigo" defined as a term, but it doesn't show up b/c of the carriage return.

This is just a javascript copy/paste, so yes, you'd have to delete it.  I don't know how events would cascade if other js manipulated it.

**Comment by Mycheze** [comment]
Yes, exactly. You can't just have characters that can't be used in the field?

**Comment by jzohrab** [comment]
> You can't just have characters that can't be used in the field?

No idea :-P -- at least I don't know how to, especially with a paste, and with the other js events.  Maybe someone smart can do it -- at any rate, it's in the backlog now.

**Comment by M-Biggles** [comment]
Just a quick questions, is the trick to remove different types of whitespace characters which lead or trail the term (allowing internal ones to be preserved for multi-part terms)?

**Comment by jzohrab** [comment]
There may be a javascript thing that could monitor the text input on change and strip out cruft ... oh but this would be in the tagify library, just remembered that, and maybe there's a way to do it with that library.  But part of me says that this is code that shouldn't be written, because it's a copy issue.
